asdf_astropy/converters/fits/tests/test_fitswcs.py

```diff
@@ -0,0 +1,80 @@
+import asdf
+import numpy as np
+import pytest
+from astropy import wcs
+
+
+def create_sip_distortion_wcs():
```

Comment:
Should I create a separate fixture for the WCS test cases, or will the existing ones work?